README.md

```diff
@@ -1,135 +1,25 @@
-Anaconda Recipes
-================
+**This repository is deprecated.**
 
-This is a Continuum managed repository for recipes used in the
-free [Anaconda Python distribution](https://www.continuum.io/downloads).
+Recipes used in the free 
+[Anaconda Python distribution](https://www.continuum.io/downloads) are hosted 
+in the Anaconda, Inc managed 
+[AnacondaRecipes/aggregate](https://github.com/AnacondaRecipes/aggregate)
```

Comment:
I think it would be better to point to the organization, rather than aggregate.  Aggregate would be better if the submodules were actual links to the repos they point at, but because that doesn't work, I think the organization is better.